SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp

```diff
@@ -353,31 +353,31 @@ CNumerics::ResidualType<> CSource_NEMO::ComputeAxisymmetric(const CConfig *confi
     for (iSpecies=0; iSpecies<nSpecies; iSpecies++)
       Mass += V_i[iSpecies]*Ms[iSpecies];
 
-    su2double heat_capacity_cp_i   = V_i[RHOCVTR_INDEX]+Ru/Mass;
+    su2double heat_capacity_cp_i   = V_i[RHOCVTR_INDEX]/rho + Ru/Mass;
     su2double total_viscosity_i    = Laminar_Viscosity_i + Eddy_Viscosity_i;
     su2double total_conductivity_i = ktr + kve + heat_capacity_cp_i*Eddy_Viscosity_i/Prandtl_Turb;
     su2double u                    = V_i[VEL_INDEX];
     su2double v                    = V_i[VEL_INDEX+1];
-    su2double qy_ve                = kve*GV[TVE_INDEX][1];
+    su2double qy_t                 = -total_conductivity_i*GV[T_INDEX][1];
+    su2double qy_ve                = -kve*GV[TVE_INDEX][1];
 
     /*--- Enthalpy and vib-el energy transport due to y-direction diffusion---*/
     for (iSpecies = 0; iSpecies < nHeavy; iSpecies++) {
-      sumJhs_y  += (rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * hs[iSpecies];
-      sumJeve_y += (rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * eve_i[iSpecies];
+      sumJhs_y  += -(rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * hs[iSpecies];
+      sumJeve_y += -(rho*Ds[iSpecies]*GV[RHOS_INDEX+iSpecies][1] - V_i[RHOS_INDEX+iSpecies]*Vector[1]) * eve_i[iSpecies];
```

Comment:
Have a look at #1312, I think there was a logic error here with how "Vector" is used which I tried to fix.